ironfish-rust/src/note.rs

```diff
@@ -273,12 +273,12 @@ impl<'a> Note {
 
     /// Computes the note commitment, returning the full point.
     fn commitment_full_point(&self) -> jubjub::SubgroupPoint {
-        // TODO: @yajun replace this with commitment_full_point_new() that takes sender address
-        commitment_full_point(
```

Comment:
@ygao76 we should clean this up after this PR, we will go back and get rid of `commitment_full_point` and refactor the `commitment_full_point_new` to have that name

Reply:
yep, thats the plan